deepchem/models/torch_models/layers.py

```diff
@@ -432,6 +432,107 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return x * norm
 
 
+class ActNorm(nn.Module):
+    """
+    PINO Activation normalization layer that applies learnable scaling and bias.
+
+    This layer is a per-channel affine normalization used by the
+    Physics-Informed Neural Operator (PINO) to stabilize spectral
+    block outputs. It applies the learnable transformation:
+
+        y = scale * x + bias
```

Comment:
Does this render correct? You may want to try rendering the docs. I am not sure if this equation displays